core/src/main/python/synapse/ml/explainers/ICETransformer.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+from synapse.ml.explainers._ICETransformer import _ICETransformer
+from pyspark.ml.common import inherit_doc
+from pyspark import SparkContext
+
+@inherit_doc
+class ICETransformer(_ICETransformer):
+    def setCategoricalFeatures(self, value):
```

Comment:
I think these should be auto-genned by the codegen system

Reply:
We discovered the same via python unit test failures. Elena will redo this part in the next commit.